static/app/components/searchQueryBuilder/valueCombobox.tsx

```diff
@@ -346,6 +347,8 @@ export function SearchQueryBuilderValueCombobox({
         onKeyDown={onKeyDown}
         autoFocus
         maxOptions={50}
+        openOnFocus
+        shouldCloseOnInteractOutside={el => el !== ref.current}
```

Comment:
Needed to add this, otherwise clicking the within the editing area would close the menu